tests/misc_env/push_cosbench_workload.py

```diff
@@ -86,7 +115,7 @@ def prepare_fill_workload(ceph_cluster, client, rgw, controller, config):
     # using the average size to find number of objects
     objects_count = math.floor(bytes_to_fill * 100 / (40456 * 1024))
```

Comment:
this average size is calculated only for fill workload hardcoded sizes
this may change for hybrid workload
need to make the calculation part dynamic
or to take object count from config, this part can be avoided in that case
both are enhancements to be done through coming PR's